sdk/src/azure/iot/az_iot_provisioning_client.c

```diff
@@ -18,6 +18,9 @@ static const az_span str_put_iotdps_register
 static const az_span str_get_iotdps_get_operationstatus
     = AZ_SPAN_LITERAL_FROM_STR("GET/iotdps-get-operationstatus/?$rid=1&operationId=");
 
+static const az_span prov_registration_id_label = AZ_SPAN_LITERAL_FROM_STR("registrationId");
+static const az_span prov_payload_label = AZ_SPAN_LITERAL_FROM_STR("payload");
```

Comment:
nit: Is there a doc that defines these string literals? Might be a good idea to link to them as a source comment.